databroker_pack/_pack.py

```diff
@@ -120,9 +120,10 @@ def export_uids(
                     root_map=source_catalog.root_map,
                     serializer_class=serializer_class,
                 )
+                file_uids[uid] = []
                 for root, set_ in files.items():
                     accumulated_files[root].update(set_)
-                    file_uids[uid] = [(root, filename) for filename in set_]
+                    file_uids[uid].extend([(root, filename) for filename in set_])
```

Comment:
This saves one iteration through the items.
```suggestion
                    file_uids[uid].extend((root, filename) for filename in set_)
```